frontend/apps/marketing/src/components/common/constants/index.ts

```diff
@@ -6,3 +6,11 @@ export const externalLinkIconProps: FontAwesomeV6IconProps = {
   iconName: 'up-right-from-square',
   iconStyle: 'solid',
 };
+
+export const FONT_SIZE = {
+  xxs: '0.625rem', // 10px
+  xs: '0.75rem', // 12px
+  s: '0.875rem', // 14px
+  m: '1rem', // 16px
+  l: '1.25rem', // 20px
+};
```

Comment:
These can be used with the Paragraph and Link components when those are refactored.